### PR TITLE
Stix2.0 parser

### PR DESCRIPTION
**Pull request by jochman**
## Status
In Progress

## Related Issues
Enhancement: https://github.com/demisto/etc/issues/15321

## Description
The script is parsing STIX2.0/1 indicators into Demisto Indicators.

## Screenshots
Paste here any images that will help the reviewer

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------

## Required version of Demisto
x.x.x

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Additional changes
Describe additional changes done, for example adding a function to common server.

### CODE REVIEW

**Comment by jochman** [comment]
tested with both stix2.0 files and json file.
fully bc.

File: Scripts/script-StixParser.yml [hunk 1]
```diff
@@ -3,66 +3,323 @@ commonfields:
...
+
+      Returns:
+          json object if succeed
+          False if failed
+      """
+      try:
+          js = json.loads(string)
+          return js
+      except ValueError:
+          return False
```
Comment (liorblob) [inline Scripts/script-StixParser.yml]: Maybe return `None` instead?

File: Scripts/script-StixParser.yml [hunk 2]
```diff
@@ -3,66 +3,323 @@ commonfields:
...
+  def get_score(description):
+      """Getting score from description field in STIX2
+
+      Args:
+          description: (str)
+
+      Returns:
+          str: `IMPACT` field
+      """
+      if description and isinstance(description, str):
```
Comment (liorblob) [inline Scripts/script-StixParser.yml]: this won't work if it's unicode on python 2.7, should be `basestring`

File: Scripts/script-StixParser.yml [hunk 3]
```diff
@@ -3,66 +3,323 @@ commonfields:
...
+          "Email": list(),
+          "URL": list(),
+      }
+      # Check if is it's
+      if isinstance(data, dict) and data.get("objects"):
+          # Create objects
+          objects = data.get("objects")
+
+          # Use regex to extract indicators
+          for obj in objects:
```
Comment (liorblob) [inline Scripts/script-StixParser.yml]: Maybe check if objects is list?

File: Scripts/script-StixParser.yml (hunk 1 above)
Comment (jochman) [reply Scripts/script-StixParser.yml]: It'll result exactly the same behavior. but fine

**Pull request merged**